tools/eslint.sh

```diff
@@ -6,6 +6,9 @@
 # shellcheck source=./tools/_functions.sh
 source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
 
+# Run eslint as a pre-commit hook
+run_as_precommit "./node_modules/.bin/eslint" "$@"
```

Comment:
```suggestion
run_as_precommit "npx eslint" "$@"
```
Is there a difference when calling it with the exact path compared to using npx? If not I'd prefer to call it like in line 14 for consistency

Reply:
Yes, there can be a difference. If node_modules is not in place, npx prettier will fetch and run the latest version of Prettier from the npm registry. This is a rare scenario, but if it happens, it could cause inconsistencies and be confusing.

![Screenshot 2024-07-15 at 18 23 40](https://github.com/user-attachments/assets/e7faba50-55ea-4329-8ebf-11f5394f8f25)

Reply:
Would pinning the version like `npx eslint@v9.7.0` remove those inconsistencies?